src/Exceptions/BadRequest.php

```diff
@@ -2,6 +2,6 @@
 
 namespace SevenShores\Hubspot\Exceptions;
 
-class BadRequest extends \Exception
+class BadRequest extends HubspotException
```

Comment:
This isn't a BC break as they both extend `\Exception`

Reply:
Ok I see